examples/web/src/routes/__root.tsx

```
@@ -6,21 +6,63 @@ import {
 	Outlet,
 	createRootRouteWithContext,
 	useRouterState,
+	Scripts,
+	Link,
 } from "@tanstack/react-router";
-import "../index.css";
+import "@/index.css";
 
-export interface RouterAppContext {}
+export interface RouterAppContext { }
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)